catalog/spatial/csw/spatial-csw-source-common/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,26 @@
+<!--
```

Comment:
Might be something we consider for the follow on ticket or just a separate ticket all together. This is a carry over from the original CswSource testing.